{"uid":"c51568a268352e16","name":"test_precision[Test for Nodle Parachain]","fullName":"tests.test_network_parameters.TestPrecision#test_precision","historyId":"78b6d2eb66b8dd41b1dbcf7e800907df","time":{"start":1669783288661,"stop":1669783289089,"duration":428},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Nodle Parachain","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7f3f2608edc0>\nchain = <tests.utils.chain_model.Chain object at 0x7f3f26102400>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f3f26102400>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Nodle Parachain\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Nodle Parachain","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7f3f2608edc0>\nchain = <tests.utils.chain_model.Chain object at 0x7f3f26102400>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f3f26102400>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Nodle Parachain\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"a96ff4ba4bfa4de7","name":"stdout","source":"a96ff4ba4bfa4de7.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestPrecision"},{"name":"host","value":"fv-az426-282"},{"name":"thread","value":"1632-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f3f26102400>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"c51568a268352e16.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f3f26102400>"]}